person.rb

```diff
@@ -13,12 +13,16 @@ def correct_name
 class Person < Nameable
```

Comment:
- [ ] Kindly add the attr_accessor for `age`, and `rental` on top of the class.

- [ ] Kindly add the method for the association between person and book.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```